RoddyCore/test/de/dkfz/roddy/config/FilenamePatternHelperTest.groovy

```diff
@@ -28,10 +44,13 @@ public class FilenamePatternHelperTest {
     }
 
     @Test
-    public void testExtractCommandsWithUnclosedLiterals() {
-        String commandID = '\${jobParameter';
+    void testExtractCommandsWithUnclosedLiterals() {
+        String commandID = '\${jobParameter'
         String temp = '/a/real/string/${jobParameter,name="test"}/${jobParameter,name="test2"/a_filename.txt'
-        assert false
+        def result = FilenamePatternHelper.extractCommands(mockupContext, commandID, temp)
+        assert result
+        // TODO What should happen?
```

Comment:
You can only test, where you have a precise understanding of what should happen. I don't want to remove the test, just because I don't have a clear idea on this now. I'd rather make it assert false then.